docs/Data_correction_documentary_cleared.ipynb

```diff
@@ -0,0 +1,299 @@
+    "This general method for time-dependent $^3$He-cells can be adapted for other use-cases such as using supermirrors or in-situ polarized ^3He-cells, see \"Note\" at the end of the page. \n",
+    "\n",
+    "First, the created test-data needs to be readout and grouped by purpose, e.g., if it describes a sample-run, a direct-beam-run, or a background measurement without cells and sample. All data will be reduced using the normalization term described in https://scipp.github.io/ess/techniques/sans/sans-beam-center-finder.html. Second, for the calculation of time-dependent $P_{^3He}(t)$, the ROI's of the direct-beam region and background region have to be defined (USING BEAMCENTER FINDER OPTION OF SANS2D?). Third, we will describe the data correction workflow with a calculation of $O$, $P_{^3He}(t)$, and $T^{\\pm}$ of the polarizer and analyzer, and the application on the sample-data.\n",
+    "\n",
+    "To facilitate an in-situ data correction during the experiments, we have implemented two different approaches: (i) a rough data-correction based on a $^3$He-cell opacity calculated by its cell parameters and filling pressure (\"workflow 1\") which can be used in-situ during the experiment, and (ii) a precise calculation based on the calculation of opacity by a direct-beam measurement on the depolarized cell at its end of lifetime (i.e., end of cell-usage for this experiment) (\"workflow 2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "405b0f3e-67ab-43d8-80de-566a1b93785a",
+   "metadata": {},
+   "source": [
+    "### Data correction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "add255a3-3222-444a-b2e6-b8cef3a8b2db",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "#### Calculation of $^3$He opacity \n",
+    "\n",
+    "$\\mathbf{Workflow 1}$:<br>\n",
+    "The $^3$He opacity is a cell-specific parameter and can vary after each filling process. It is given by the cell pressure $p$ at 20C, the path length $l$ of neutron through the cell, and the neutron wavelength $\\lambda$ (Ref 1):\n",
+    "\n",
+    "$$\n",
+    "O^{cell}(\\lambda)= (0.0733 \\cdot p^{cell} \\cdot l^{cell} )\\lambda\n",
+    "$$\n",
+    "\n",
+    "with a factor 0.0733 arising from the number density of $^3$He gas and the absorption cross section of 1$\\mathring A$ neutrons by $^3$He. It can be seen that the opacity depends linearly on the neutron wavelength $\\lambda$. To calculate one value of $O(\\lambda)$ that fits to all wavelength from the depolarized direct-beam data $I^{DB, depol, cell}$, we will fit the wavelength-independent part $O^{0, cell}$ of $O(\\lambda)=O^{0, cell}\\cdot\\lambda$ in above equation. Afterwards, the wavelength-dependence can be retrieved by setting \n",
```

Comment:
What do you mean by 'we will fit the wavelength-independent part'? Can't you just measure `l` and `p` and compute O^0?

EDIT: After reading on, it seems that everything up to here is the same for both workflows. So why is it in the 'Workflow 1' section?

Reply:
1) we did not divide the workflows 1 and 2 into parts defined in the beginning of the script and end of the script, but into how they are physically derived. 
In workflow 1, the opacity will be acquired by setting O=p l lambda = O_0 lambda, whereas in workflow 2 it will be acquired by the direct beam through the depolarized cell and fitting O_0 to the given function. 
--> the workflows are the same everywhere except for how to derive O_0

2) for workflow 1 with setting O=p l lambda = O_0 lambda:
--> yes, O_0 is set by predefined p and l, that can be inserted by the sample environemtn team prior to the user measurement and will be just read-in values. still, O = O_0 lambda is wavelength dependent, and the lambda-dependency in how its written here is then treated the same as for workflow 2.
--> but for workflow 2, this does not work, O_0 has to be fit to the given function in workflow 2

Reply:
So $O(\\lambda)=O^{0, cell}\\cdot\\lambda$ is used in both workflows? Then why is it listed in the 'workflow 1' section? Can you move the common equations and text out of the workflow sections and cleanly separate the workflows into sections? This would make it easier to understand how things relate to each other.

Also, 'workflow 2' says
> A more precise data correction is based on a measure of $p$ by a direct-beam measurement through the depolarized cell

Do you really measure $p$? It sounds more like you don't use the parametrisation with $p$ and $l$ in workflow 2 but treat $O^0$ as a fundamental parameter.

Reply:
Thanks for this comment, it is a good idea to divide these information in a better way! I will upload a new version. I am also sorry for confusion, I meant $P(t)$ here, not p (pressure of cell). I have rewritten it.